Standard/Events/GameEvent.cs

```diff
@@ -6,20 +6,20 @@ namespace CyberneticStudios.SOFramework
     /// Base class of all Event Channels. Event channels are scriptable objects that raise events and can be listened
     /// </summary>
     /// <typeparam name="T"></typeparam>
-    public abstract class EventChannelSO<T> : ScriptableObject
+    public abstract class GameEvent<T> : ScriptableObject
     {
 
 #if UNITY_EDITOR
         [Multiline]
         public string DeveloperDescription = "";
 #endif
 
-        public System.Action<T> OnEventRaised;
-        [SerializeField] private DebugInformation debugInformation;
+        public System.Action<T> OnGameEventRaised;
+        [SerializeField] DebugInformation debugInformation;
```

Comment:
I prefer adding private before variables even if they don't need it, so it is visibly clear and cleaner when compared to public members